vocab_files/attribute_concepts/equipment-dimensions.ttl

```diff
@@ -11,9 +11,9 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision
 making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "The dimensions of the equiments used during a pasive- targeted fauna survey. " ;
+    skos:definition "The dimensions of the equipments used during a passive- targeted fauna survey." ;
```

Comment:
the definition should be "The dimension of the equipment used during a passive-targeted fauna survey."